admin/src/app/admin/navigation/navigation-list-item/navigation-list-item.component.ts

```diff
@@ -7,6 +7,7 @@ import { Component, OnInit, Input } from '@angular/core';
 })
 export class NavigationListItemComponent implements OnInit {
   @Input() icon: string;
+  @Input() isExpanded: string;
```

Comment:
`isExpanded` should be a boolean

Reply:
Fixed in commit https://github.com/oddbit/tanam/pull/34/commits/f20f90e97f0fbdfdf949a00cc35167170f61f163